covid-dashboard/src/js/getApiData.js

```diff
@@ -0,0 +1,81 @@
+// for table, list with flags
+async function getCountriesAndGlobalInfo() {
+  const urlCountries = 'https://disease.sh/v3/covid-19/countries';
+  const urlGlobal = 'https://disease.sh/v3/covid-19/all';
+  const responseCountries = await fetch(urlCountries);
+  const responseGlobal = await fetch(urlGlobal);
+  const countryList = await responseCountries.json();
+  const global = await responseGlobal.json();
+  const countriesInfo = countryList.reduce((accum, {
+    country,
+    countryInfo,
+    cases,
+    todayCases,
+    deaths,
+    todayDeaths,
+    recovered,
+    todayRecovered,
+    population,
+    updated,
+  }) => {
+    // eslint-disable-next-line no-param-reassign
+    accum[country] = {
+      flag: countryInfo.flag,
+      cases,
+      todayCases,
+      deaths,
+      todayDeaths,
+      recovered,
+      todayRecovered,
+      population,
+      updated,
+    };
+    return accum;
+  }, {});
+  const {
+    cases,
+    todayCases,
+    deaths,
+    todayDeaths,
+    recovered,
+    todayRecovered,
+    population,
+    updated,
+  } = global;
+  countriesInfo.Global = {
+    cases,
+    todayCases,
+    deaths,
+    todayDeaths,
+    recovered,
+    todayRecovered,
+    population,
+    updated,
+  };
+  console.log(JSON.stringify(countriesInfo, null, 2)); // just for control
+  return countriesInfo;
```

Comment:
It's better to split such a big function to smaller ones.
fetchContriesData, fetchGlobalData, getContriesData, get Global Data, getContriesAndGlobalData.
The first two just fetch data from different endpoints, the second two map to appropriate data format, and the last one aggregates both.